stdlib/LinearAlgebra/src/adjtrans.jl

```diff
@@ -138,6 +138,10 @@ IndexStyle(::Type{<:AdjOrTransAbsMat}) = IndexCartesian()
 convert(::Type{Adjoint{T,S}}, A::Adjoint) where {T,S} = Adjoint{T,S}(convert(S, A.parent))
 convert(::Type{Transpose{T,S}}, A::Transpose) where {T,S} = Transpose{T,S}(convert(S, A.parent))
 
+# Strides for transposed strided arrays — but only if the elements are actually stored in memory
+Base.strides(A::Adjoint{<:Real, <:StridedArray}) = reverse(strides(A.parent))
```

Comment:
As noted in the comments `reverse` won't quite do it. But you need the array available because it requires a call to `length`, so here I'd just specialize separately on `StridedVector` and `StridedMatrix`.

If one really wants to be defensive, this may be worth noting:
```julia
julia> Adjoint(rand(3,3,3));  # no error
```